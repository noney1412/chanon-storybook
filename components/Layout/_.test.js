import C from './'

test('Post component should render the title not as a link', () => {
  const component = shallow(<C />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})


