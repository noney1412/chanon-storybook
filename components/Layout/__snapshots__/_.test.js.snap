// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post component should render the title not as a link 1`] = `
<div>
  Layout
</div>
`;
